mantidproject/mantid
# Lr71 subtract bkgd

**Description of work.**

In Mantid algorithm `LRSubtractAverageBackground,`, add a new input property 'WeightedBackground' and set default to False 

Keep all the callers of LRSubtractAverageBackground  the current status, i.e., to set WeightedBackground True.

By this, there shall be no change in testing result.

This is the implementation of [ORNL REFL #76](https://code.ornl.gov/sns-hfir-scse/reflectometry/reflectometry/-/issues/76)

**To test:**

- Make sure all the builds are passed.
- Visual check the changes.

Fixes #xxxx. 







---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
